admin_guide/iptables.adoc

```diff
@@ -11,18 +11,76 @@
-recommended that the rules are saved and restored. 
+IMPORTANT: You must make sure that whatever changes you make to the iptables configuration on a node
+do not impact the operation of {product-title} and Docker. Also, changes will often need to be made on
+all nodes in the cluster. Be careful, iptables is not designed to have multiple concurrent users and it
+is very easy to break {product-title} and Docker networking.
+
+The chains, order of the chains and rules in the kernel iptables must be properly set up on each node
+in the cluster for {product-title} and Docker networking to work properly. There are several tools and
+services that are commonly used in the system that interact with the kernel iptables and can
+accidentally impact {product-title} and Docker.
+
+=== iptables
+`iptables` is a low level tool that can be used to set up, maintain, and inspect the tables of IPv4
+packet filter rules in the Linux kernel.
+
+WARNING: This is a very powerful tool that can manage all aspects of the kernel iptables. It is very
+easy to break networking in difficult to debug ways so be careful.
+
+Independent of other use, such as a firewall, {product-title} and Docker manage chains in some of the tables.
+The chains are inserted in specific order and the rules are specific to their needs.
+
+IMPORTANT: `iptables --flush [chain]` can remove key required configuration. Be very careful when using this.
+
+When making changes consider limiting your changes to the OS_FIREWALL_ALLOW chain in the filter table.
+
+=== iptables.service
+The iptables service supports a local network firewall. It assumes total control of the iptables
+configuration and when it starts it flushes and restores the complete iptables configuration.
+The restored rules are from its configuration file (/etc/sysconfig/iptables).
+The config file is not kept up to date during operation so every restart the dynamically added rules are lost.
+
+WARNING: Stopping and starting the iptables.service will destroy configuration that is required by
+{product-title} and Docker. {product-title} and Docker are not notified of the change.
+
+WARNING: Use of iptables.service is not recommended in conjunction with {product-title} or Docker.
+
+Iptables.service can be disabled and masked to prevent accidental execution:
+
+----
+# systemctl disable iptables.service
+# systemctl mask iptables.service
+----
+
+If you need to run iptables.service, keep a limited configuration in the config file and
+rely on Docker and openshift to install their needed rules.
+
+The iptables.service configuration is loaded from:
+
+----
+/etc/sysconfig/iptables
+----
+
+To make permanent rules changes edit the changes into this file. Do not include Docker or {product-title} rules.
+
+After iptables.service is started on a node, Docker must be started or restarted and atomic-openshift-node.service
+must be restarted to reconstruct the needed iptables configuration.
```

Comment:
Sorry, my earlier comment here was ambiguous... this should be: After iptables.service is started or restarted on a node, Docker and atomic-openshift-node.service must be restarted to reconstruct the needed iptables configuration.